trl/models/modeling_base.py

```diff
@@ -68,7 +66,7 @@ class PreTrainedModelWrapper(nn.Module):
             The list of arguments that are supported by the wrapper class.
     """
     transformers_parent_class = None
-    supported_args = None
+    supported_args = ("score_module", "supports_rm_adapter", "rm_adapter_name")
```

Comment:
Do you know if these are used? It seems we always overwrite them: https://github.com/huggingface/trl/blob/main/trl/models/modeling_value_head.py#L90

Reply:
I made this change so it would be a bit more obvious the args being passed in. I'm happy to change it back if you'd like

I actually don't think we use `supported_args` anywhere. I assumed it would be a future feature for some sort of argparsing. To be more in line with `transformers` we would probably want a `config` class instead of supported args anyways so it isn't a big deal either way.

Reply:
OK I see, thanks! 
Yes it is not a big deal, technically I have set it to `None` so that the `PreTrainedModelWrapper` can't be used as it is (I should have set that class as an abstract class). Yes it would be great if you can change this back then we can merge I think

Reply:
Done!